lib/vrl/tests/src/docs.rs

```diff
@@ -0,0 +1,162 @@
+use serde::Deserialize;
+use std::collections::{BTreeMap, HashMap};
+use std::fs;
+use std::process::Command;
+// use vrl::Value;
+use serde_json::{Map, Value};
+
+use crate::Test;
+
+/// A list of function examples that should be skipped in the test run.
+///
+/// This mostly consists of functions that have a non-deterministic result.
+const SKIP_FUNCTION_EXAMPLES: &[&str] = &[
+    "uuid_v4",
+    "strip_ansi_escape_codes",
+    "get_hostname",
+    "now",
+    "get_env_var",
+];
```

Comment:
We could add a field (e.g. `skip_test`) in cue so that it doesn't require recompilation of source code.

Reply:
Good call. Let's bench that for a future iteration, as I don't think there will be much churn in this list going forward.